### PR TITLE
[docs] company name change from Continuum to Anaconda

### PR DESCRIPTION
**Pull request by electronwill**
[docs] company name change from Continuum to Anaconda

### CODE REVIEW

**Comment by electronwill** [comment]
@postelrich , would you please merge this one or let me know what else I should do to make this ready to merge?

**Comment by electronwill** [comment]
Thank you!

File: docs/source/dev_workflow.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ Developer Workflow
 This page describes how to install and improve the development version of Blaze.
 
 If this documentation isn't sufficiently clear or if you have other questions
-then please email blaze-dev@continuum.io.
+then please email blaze-dev@anaconda.com.
```
Comment (llllllllll) [inline docs/source/dev_workflow.rst]: does this address actually reach any developers?

Comment (electronwill) [reply docs/source/dev_workflow.rst]: I think this is the list: https://groups.google.com/a/anaconda.com/forum/?hl=en#!members/blaze-dev

Comment (electronwill) [reply docs/source/dev_workflow.rst]: Looks like 5 threads this year, most recently in July. https://groups.google.com/a/anaconda.com/forum/#!forum/blaze-dev

**Pull request merged**